README.rst

```diff
@@ -0,0 +1,36 @@
+=============================
+Haskell SPAKE2 implementation
+=============================
+
+Implementation of SPAKE2 key exchange protocol.
+
+Status
+======
+
+Doesn't actually work. We are in the middle of implementing it.
```

Comment:
I looked this up. Seems like a great idea, but I've never tried it before so I'll probably get it wrong. Happy to try it though.